docs/diagrams/LogicClassDiagram.puml

```diff
@@ -27,8 +27,8 @@ Class HiddenOutside #FFFFFF
 HiddenOutside ..> Logic
 
 LogicManager .right.|> Logic
-LogicManager -right->"1" AddressBookParser
-AddressBookParser ..> XYZCommand : creates >
+LogicManager -right->"1" CompanyListParser
+CompanyListParser ..> XYZCommand : creates >
 
```

Comment:
Not necessary but we can consider having a high level overview of parser and command class interaction within logic, and then breakdown the parser and command classes individually using two separate diagrams.

E.g. Can refer to the Model classes for reference

Reply:
Okay this sounds good 👍 I'll try to write something up for this.